snippets/language-asciidoc.cson

```diff
@@ -44,10 +44,10 @@
       ----
       $0
       '''
-  'Code Block':
-    description: 'Code Block'
+  'Source Block':
+    description: 'Source block'
     descriptionMoreURL: 'http://asciidoctor.org/docs/user-manual/#listing-blocks'
-    prefix: '--s'
+    prefix: '``'
```

Comment:
It's first version, we can change before, if you want I remove this part for now ?